### DISCUSSION TITLE
Proposal Mocaccino logo

### DISCUSSION THREAD

**Discussion by vargux** [mocaccinoOS/mocaccino#80] [Show and tell] [answered]
Hi,
This is only a proposal Mocaccino logo.
![MocaccinoOS](https://user-images.githubusercontent.com/879795/172040393-e35c882a-a99e-4b1d-833c-1bfde5c28b67.svg)

1: A simple idea.
2: With a Gentto an Sabayon Linux logo inside.
3: Same of 2, but more "clean".

I've made using Inkscape.
Greetings.
VARGUX

**Comment by joostruis** [comment]
I like the idea. We could maybe even use the idea for a Plymouth theme where we start with the first image and the 2 logo's transition in.

**Reply by joostruis** [reply]
Or maybe better, since Sabayon and Gentoo combined form Mocaccino. Start with the 2 logo's that morph into what results as the first image.

**Reply by 01fiore** [reply]
+1

**Comment by mudler** [comment]
That looks really nice! Thank you so much! I'd go for using 1. as a baseline, and as @joostruis suggests an animation would make a perfect fit for a Plymouth splash screen

**Comment by msdobrescu** [comment]
I love it, but what is the relation of this work to the one here https://www.facebook.com/arabiclanguaguniversity/posts/you-already-know-arabicwhat-is-this/1082136435328100/? The reason of the question is to avoid copyright issues.
I personally would like (1) with the Gentoo instead of the heart, but with the perspective of the heart of (1).
I'd also like less thick outlines (but this is my preference).

**Reply by vargux** [reply]
The main image (the cup) it's from wikipedia... https://commons.wikimedia.org/wiki/File:Applications-ristretto.svg#mw-jump-to-license

Maybe the Gentoo logo is not GPL...
The image number 1 it's OK speaking about it licence.

These were just ideas... they can and should be improved...

Greeting!

**Reply by joostruis** [reply]
Gentoo logo license is CC BY-SA 2.5. https://creativecommons.org/licenses/by-sa/2.5/
Sabayon logo would be no problem to use either.

**Comment by vargux** [comment, marked as answer]
Hi!...
A very light timeline version of the logo with is [here](https://commons.wikimedia.org/wiki/File:MocaccinoOS.svg)

**Reply by vargux** [reply]
I'm a Wikipedian, that's why I uploaded the image there.

Please take a look to my gallery [here](https://commons.wikimedia.org/wiki/User:VARGUX).

**Reply by vargux** [reply]
Logo versions:

![MocaccinoOS logo](https://user-images.githubusercontent.com/879795/172494382-a3a6d561-3d77-4fc5-afd3-f95c6a819f6d.svg)

![MocaccinoOS logo 2](https://user-images.githubusercontent.com/879795/172494387-a146d28a-e361-4e59-8e93-e060b3677a56.svg)

![MocaccinoOS logo 3](https://user-images.githubusercontent.com/879795/172494394-636f9c14-50a8-452a-b9bf-05e29f767e18.svg)

**Reply by msdobrescu** [reply]
Hi, this is nice! Look [here](https://www.gentoo.org/inside-gentoo/artwork/gentoo-logo.html), this is the Gentoo logo (g).
Also [here](https://www.portagefilelist.de/site/query/file/?do#result), this is the Portage logo (p).
Above, I see on the cup, the "p" reversed, right?
Here we have the rotated "g":
![image](https://user-images.githubusercontent.com/7279637/172527192-38dc23b1-f9ee-4b99-b220-80c6488cd1f1.png)
Here the rotated "p":
![image](https://user-images.githubusercontent.com/7279637/172527303-97a157ff-7912-4ac3-8c79-0c26d49c974b.png)
What is your intention?

**Reply by vargux** [reply]
The idea is to use some symbol of where Mocaccino comes from... And also being similar to a shape I've seen on some real mocaccino cups.

**Reply by 01fiore** [reply]
Beautiful, my favorites, 1) logo of the distribution. 2) icon of the start menu of the different offices, KDE Gnome XFCE etc. 

Tchao.

**Reply by 01fiore** [reply]
Sorry, mes favoris 1 & 3.

**Comment by msdobrescu** [comment]
Here is my original draft of a latte art:
<img src="https://user-images.githubusercontent.com/7279637/172527586-c88ceadc-322e-4a76-a42f-1d97bcb84232.svg" width="150"/>
I've tried this, I let you guess why I didn't proposed this:
<img src="https://user-images.githubusercontent.com/7279637/172527898-36d4e99f-1026-40a7-8d62-28368356c1ce.svg" width="150"/>

**Reply by vargux** [reply]
Both logos are fine for me...because I'm not the one to decide which logo is more suitable. 
Those who are developing this distro are the ones who must decide.

**Reply by msdobrescu** [reply]
Me neither. I just maintain C-R. Maybe we can reuse them or not. I am not a designer either. Just having fun with Inkscape. Those are only suggestions. My personal preference is realism or abstract. I'd keep the original outlines (for abstract) or drop them completely (for realism - but would need a little rework).
Are you a designer? Can you draw an astronaut like the one on the default wallpaper?

**Reply by msdobrescu** [reply]
And yes! I like your work!

**Reply by vargux** [reply]
which default wallpaper are you talking about? :)

**Reply by vargux** [reply]
> Me neither. I just maintain C-R. Maybe we can reuse them or not. I am not a designer either. Just having fun with Inkscape. Those are only suggestions. My personal preference is realism or abstract. I'd keep the original outlines (for abstract) or drop them completely (for realism - but would need a little rework). Are you a designer? **Can you draw an astronaut like the one on the default wallpaper?**

https://www.mocaccino.org/featured-background_hu2226c7009d5c201991b3c3f4c09a60eb_1455824_960x540_fill_q75_catmullrom_top.jpg
¿This one ?

**Reply by msdobrescu** [reply]
Yes, but not exactly like that. Just an astronaut. Would be a mascot.

**Reply by germe-deb** [reply]
I really love the first one!!!

And It can fit perfectly on my icon pack

![Captura de pantalla de 2022-06-10 22-50-20](https://user-images.githubusercontent.com/95252500/173168103-d90080da-317e-4ec6-aa11-654478bb2fdd.png)
![mocaccinoOS icon](https://user-images.githubusercontent.com/95252500/173168107-e65bc5a5-c7a4-4fb2-9979-b3161b7c19fd.svg)

**Reply by germe-deb** [reply]
also, a plymouth animation can be very easy to design. I think on a cup of mocaccino waving a bit, like this four frames with some interpolation that makes a feeling of movement:

![Captura de pantalla de 2022-06-10 22-57-34](https://user-images.githubusercontent.com/95252500/173168331-bcfdcbdd-5e51-462d-8c02-6786426e7b1e.png)
(The first one is static, but the others are a bit rotated and moved)
![mocaccinoOS icon - round - plymouth](https://user-images.githubusercontent.com/95252500/173168332-a6e71f90-a359-4c24-9bf3-51418f3d7c26.svg)

**Reply by msdobrescu** [reply]
This makes me wonder how to make a simulation of a latte art creation.

**Comment by joostruis** [comment]
So about the Plymouth idea. What needs to be done is create a zip file with the images and the theme.
Currently we use the spinner theme. You can find it in /usr/share/plymouth/themes/spinner:
![image](https://user-images.githubusercontent.com/1949179/173403533-8cbf5d81-1e41-4643-9044-e201036392c0.png)

If somebody cares to do this, I will package it and test it!

**Reply by joostruis** [reply]
@germe-deb Can you do this?

**Reply by germe-deb** [reply]
> @germe-deb Can you do this?

I don't know. Send me the repo and I'll fork it with the animation

**Reply by joostruis** [reply]
I think the origional theme is here:
https://github.com/sergeysova/plymouth/tree/master/themes/spinner
